### ISSUE TITLE
InvalidKerasConfigurationException: Keras layer is missing init field

### ISSUE THREAD

**Issue by lyg5623** [deeplearning4j/deeplearning4j#3298]
#### Issue Description
it throws a InvalidKerasConfigurationException while I load a Keras model with the code below:
`ComputationGraph computationGraph= KerasModelImport.importKerasModelAndWeights("rcnn.json","rcnn.h5", false);`

rcnn.json and rcnn.h5 are model json file and weights hdf5 file export with Keras code:
`json_string = model.to_json()`
`textfile = open('rcnn.json', 'w')`
`textfile.write(json_string)`
`textfile.close()`
`model.save_weights('rcnn.h5')`
where model is a keras model created by functional API. 

the exception stack trace is:
org.deeplearning4j.nn.modelimport.keras.InvalidKerasConfigurationException: Keras layer is missing init field. For more information, see http://deeplearning4j.org/model-import-keras.
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getWeightInitFromConfig(KerasLayer.java:961)
	at org.deeplearning4j.nn.modelimport.keras.layers.KerasEmbedding.<init>(KerasEmbedding.java:74)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getKerasLayerFromConfig(KerasLayer.java:235)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperPrepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:164)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:97)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel$ModelBuilder.buildModel(KerasModel.java:576)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:177)

it seems that KerasModelImport would read a field named "init" which doesn't exist in config file because the layerConfig is:
`{class_name=Embedding, config={embeddings_initializer={class_name=RandomUniform, config={maxval=0.05, seed=null, minval=-0.05}}, name=embedding_1, dtype=int32, output_dim=100, trainable=true, embeddings_regularizer=null, input_dim=4671, mask_zero=false, embeddings_constraint=null, batch_input_shape=[null, null], activity_regularizer=null, input_length=null}, inbound_nodes=[[[input_1, 0, 0, {}]], [[input_2, 0, 0, {}]], [[input_3, 0, 0, {}]]], name=embedding_1}`

#### Version Information
version information in keras environment:
* Keras version:  2.0.3
* platform information:  debian jessie
* CUDA version: 8.0
* NVIDIA driver version: 375.26

version information in Deeplearning4j environment:
* Deeplearning4j version:  0.8.0
* platform information:  windows 7
CPU only

**Comment by jgustave** [comment]
I ran in to the same issue with an HDF5 file.

**Comment by sgiorgis** [comment]
Ι ran in to the same issue . I get this error only when i use an embedding layer as my first layer in my network . I get this error : 


SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Exception in thread "main" org.deeplearning4j.nn.modelimport.keras.InvalidKerasConfigurationException: Keras layer is missing init field. For more information, see http://deeplearning4j.org/model-import-keras.
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getWeightInitFromConfig(KerasLayer.java:961)
	at org.deeplearning4j.nn.modelimport.keras.layers.KerasEmbedding.<init>(KerasEmbedding.java:74)
	at org.deeplearning4j.nn.modelimport.keras.KerasLayer.getKerasLayerFromConfig(KerasLayer.java:235)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.helperPrepareLayers(KerasModel.java:190)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:99)
	at org.deeplearning4j.nn.modelimport.keras.KerasSequentialModel.<init>(KerasSequentialModel.java:56)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel$ModelBuilder.buildSequential(KerasModel.java:581)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasSequentialModelAndWeights(KerasModelImport.java:216)

**Comment by tomthetrainer** [comment]
Keras 2.0.x support is not ready. 

Keras 1.x support is ready. 

@lyg5623 I assume that is your issue

@sgiorgis 
What Keras version?

**Comment by sgiorgis** [comment]
@tomthetrainer  thanks you for your answer . I'm using Keras (2.0.4) . When do you think the support is going to be ready? At least for this issue?

**Comment by jgustave** [comment]
Same Keras 2.0.3.
If I just cut out the initialization from DL4J, would that buy me anything?

-JD

**Comment by lyg5623** [comment]
@tomthetrainer thank you for your reply

**Comment by tomthetrainer** [comment]
Here is the issue to add support. 

Questions on progress can be handled there, though in general our rule is "it is ready when it is ready" 

Of course adding support for Keras 2.0 is important to us, so it will not be ignored, we just have a lot going on.
 
https://github.com/deeplearning4j/deeplearning4j/issues/3129

Is it possible to work around this by rolling back to Keras 1.3 on your end?

**Comment by turambar** [comment]
Given that this appears to be related to "Keras 2," I'm going to close the issue in favor of https://github.com/deeplearning4j/deeplearning4j/issues/3129.
